### PR TITLE
feature EPL-1407: How to add Columns to a Table

### PR DESCRIPTION
**Pull request by Tomas-Vesco**

### CODE REVIEW

**Review by RomanMagnoli — APPROVED** [review]

File: mkdocs.yml [hunk 1]
```diff
@@ -316,7 +316,7 @@ nav:
         - How to Add a Field to a Window Tab: developer-guide/etendo-classic/how-to-guides/How_to_add_a_field_to_a_Window_Tab.md
 #        - How to add a masked text field: developer-guide/etendo-classic/how-to-guides/How_to_add_a_masked_text_field.md
         - How to Add a Rich Text Field and Column: developer-guide/etendo-classic/how-to-guides/How_to_add_a_rich_text_field_and_column.md
-#        - How to add Columns to a Table: developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md
+        - How to add Columns to a Table: developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md
```
Comment (valeg-etendo) [inline mkdocs.yml:319]:
```suggestion
        - How to Add Columns to a Table: developer-guide/etendo-classic/how-to-guides/how-to-add-columns-to-a-table.md
```

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+  - add columns
+  - table
+  - configuring columns
+  - application dictionary
+---
+#  How to Add Columns to a Table 
+
+##  Overview
+
+In this section, the user can find information about how to extend the table created in the [How to Create a Table](../../../developer-guide/etendo-classic/how-to-guides/How_to_create_a_Table.md) section to include additional fields.
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md:13]: @isaiasb-etendo Is this change suggestion correct?

```suggestion
In this section, the user can find information about how to extend the table created in the [How to Create a Table](how-to-create-a-table.md) section to include additional fields.
```

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
+---
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md:1]: Rename as `how-to-add-columns-to-a-table.md`

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md [hunk 4]
```diff
@@ -0,0 +1,188 @@
...
+Column name  |  Type  |  Length  |  Note  
+---|---|---|---  
+em_ht2_validto  |  DATE  |  |  Date that this salary is valid to.  
+em_ht2_payment_schedule  |  VARCHAR  |  60  |  When the salary is paid  
+em_ht2_c_salary_category_id  |  VARCHAR  |  32  |  Link to the Salary Category  
+  
+To create the above table within the database, use one of the following ALTER TABLE statements depending on the DB you are using:
+
+*PostgreSQL*
+```sql
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md:65]:
```suggestion
```sql title="SQL script"
```

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md [hunk 5]
```diff
@@ -0,0 +1,188 @@
...
+
+Column name  |  Type  |  Length  |  Note  
+---|---|---|---  
+em_ht2_validto  |  DATE  |  |  Date that this salary is valid to.  
+em_ht2_payment_schedule  |  VARCHAR  |  60  |  When the salary is paid  
+em_ht2_c_salary_category_id  |  VARCHAR  |  32  |  Link to the Salary Category  
+  
+To create the above table within the database, use one of the following ALTER TABLE statements depending on the DB you are using:
+
+*PostgreSQL*
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md:64]:
```suggestion
```

File: docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md [hunk 6]
```diff
@@ -0,0 +1,188 @@
...
+This section only lists the main important elements needed for the new module. More details can be found in the How to Create and Package a Module tutorial.
+
+  * dbprefix: *HT2*
+  * Dependencies: 
+    * Core as usual 
+    * _Openbravo Howtos_ module (>= 1.0.0) is also needed as this new module adds columns to that existing one, and so needs to ensure the first module is present. 
+
+###  Creating the columns in the database
+
+Column name  |  Type  |  Length  |  Note
```
Comment (valeg-etendo) [inline docs/developer-guide/etendo-classic/how-to-guides/How_to_add_Columns_to_a_Table.md:56]: @Tomas-Vesco Fix table formatting

**Review by valeg-etendo — CHANGES_REQUESTED** [review]

**Review by valenvivaldi — APPROVED** [review]

**Review by valeg-etendo — APPROVED** [review]

**Pull request merged**